pom.xml

```diff
@@ -106,12 +146,19 @@
 						</goals>
 						<phase>process-resources</phase>
 						<configuration>
-							<propertyFile>db.properties</propertyFile>
+							<propertyFile>liquibase.properties</propertyFile>
 							<changeLogFile>db\migration\dbchangelog.xml</changeLogFile>
 						</configuration>
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-war-plugin</artifactId>
+				<configuration>
+					<webXml>D:\Projects\bestShop\src\main\resources\web.xml</webXml>
```

Comment:
Please use cnventional path, i.e. WEB-INF/web.xml. Also use classpath: instead of absolute path.